cli/cdsctl/admin_services.go

```diff
@@ -131,3 +132,20 @@ func adminServiceGetRun(v cli.Values) error {
 	fmt.Println(string(btes))
 	return nil
 }
+
+var adminServiceDeleteCmd = cli.Command{
+	Name:  "delete",
+	Short: "Delete a CDS service from registered service",
+	Flags: []cli.Flag{
```

Comment:
it should be a variadic arg instead of of a flag

Reply:
done